docs/UserGuide.md

```diff
@@ -72,295 +82,468 @@ Information that is important to know to prevent unexpected or undefined behavio
 ## Quick Start
 ### Installation
 
-1. Ensure you have Java `11` or above installed in your computer.
+1. Ensure you have Java `11` or above installed in your computer by running the command
+    `java --version` in your terminal.
 
-1. Download the latest `SOCareers.jar` [here]([https://github.com/AY2324S1-CS2103T-T10-4/tp/releases).
+2. Download the latest version of `SOCareers.jar` [here]([https://github.com/AY2324S1-CS2103T-T10-4/tp/releases).
 
-1. Move `SOCareers.jar` to the _home folder_ which is the folder you want to store SOCareers' internal data.
-<div markdown="span" class="alert alert-info">
+3. Move `SOCareers.jar` to the home folder which is the folder you want to store SOCareers' internal data.
+<div markdown="span" class="alert alert-secondary">
 **:information_source: Information**<br>
 The home folder's contents may be changed by SOCareers.
-It is recommended to create a new empty folder to act as a home folder.
+We recommend you to create a new empty folder to act as your home folder.
 </div>
 
-1. Open a command terminal, navigate into the folder you put the jar file in using `cd` , and run the command
-`java -jar SOCareers.jar` to start the application.<br>
-   * A GUI similar to the one shown below should appear in a few seconds.
-   * Note that the app contains some sample data.<br>
-![Ui](images/Ui.png)
-
-2. Our Ui has 6 main sections:
+1. Open a command terminal.
+2. Navigate your home folder by running the command `cd FILEPATH` where `FILEPATH` is the file path to your home folder.
+3. Run the command `java -jar SOCareers.jar`  to start the application.
+    * A GUI similar to the one shown below should appear in a few seconds.
```

Comment:
say Graphical User Interface (GUI) instead? its a cs2101 thing haha

Reply:
Since I don't think we mentioned what's a GUI before

Reply:
We mentioned it in line 14